applications/bluesky.md

```diff
@@ -0,0 +1,232 @@
+	- On-chain smart contract upgradability
+	- Simplified, reusable and easy-to-read ink! code via macros 
+	- Custom pallets implemented ink! macros for balacing common logic abstration from application logic
+	- Cost effective and predictable gas fees via benchmark against Solidity
+- Choosen project idea category or your alternative category with explanation of its importance
+	- Technical showcase to domonstrate multiple benefits of ink! / WASM based smart contracts, especially as compared to Solidity / EVM based ones
+- An indication of why your team is interested in creating this project.
+	- Unlock Polkadot / Kusama / ink! ecosystem's technical strength and potentials as an smart contract alternative to Solidity / EVM 
+	- Help ink! novice, especially Solidity developers. without prior knowledge of ink! / Substrate / Polkadot
+
+### ink! Ecosystem Impact
+
+Please clearly describe how exactly your project will benefit the ecosystem. If it's infrastructure - how it's going to be applied and by who. If it's canary dapp we would you to clearly demonstrate how this code will be used by other people. If it's technical showcase we would like you to demonstrate how ink! will be promoted using your results. If it's a business case we would like to see b usiness model and future production plans. 
+
+As a techbical showcase project, ink! will be promoted competitively against Solidity/EVM based smart contracts with clear quanfitied  
+
+- Simplified, reusable and easy-to-read ink! code via macros 
+- Custom substrate pallets coupled with ink! macros to allow developers abstract common reusable logic from application logic and optimize over iterations to balance, yet still organically fused via ink!
+- Cost effective and predictable gas fees via benchmark against Solidity/EVM resulting in quantifiable metrics 
+- On-chain WASM based smart contract upgradability, as compared to EVM based smart contract upgrades on Ethereum  
+- Increase ink! adoption and future growth via competitive analysis against Solidity / EVM
+
+### Project Details
+
+Here's a list of more details of ink! smart contract, Substrate custom pallets, etc to showcase simplified smart contract code, lower gas fees and better perfromance.  
+
+- ink! Macros
+	-  ink! macros are a way to define custom syntax or code that can be reused within a smart contract. Macros in ink! are defined using the Rust programming language's macro system, which provides a powerful set of tools for code generation and metaprogramming.
+	- Code simplicity via macros: use ink! macros to simplify the code by abstracting away the complexity of the application logic. 
+	- Design a list of ink! macros for the use case to demonstrate reusable code patterns for functions, data structures, and other elements of a smart contract, ehich can help to simplify code and reduce repetition, making it easier to write and maintain complex contracts
+
+- Substrate custom pallets
```

Comment:
Do you really need it? If I understand correctly what you propose, i.e. the you want to have some logic implemented as an ink contract and as equivalent pallet then in my opinion it goes in the opposite direction than the story of the application. I mean that it's hard to believe that benchmarks of those two approaches would be comparable since contract side has additional layer of execution - wasmi, and in general the contract environment is much more constrained that the environment of a pallet. I'd suggest to compare only to solidity/EVM.

Reply:
Good point. Was not clear on what to compare.  I've revised milestone#3 step5 as below to clarify and make it optional  to focus more on comparison with Solidity instead as recommended:

Compare the gas costs between ink! smart contract with macros (including custom pallet costs) vs one without macros, use a test suite to measure the gas costs, then tune the logic placement in macros/custom pallets/smart contract code to optimize for overall gas costs